client/src/components/CreatePost/Form/ThirdSection.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Head from "./Head";
+import { SubTitle, Section, Selector } from "components/CreatePost/StyledModal";
+import SvgIcon from "components/Icon/SvgIcon";
+import downArrowSlim from "assets/icons/down-arrow-slim.svg";
+
+const Third = ({ formData }) => {
+  const { shareWith, expires } = formData;
+  return (
+    <Section>
+      <Head number={3} title="What is the visibility of your post?" />
+      <div className="buttons">
+        <SubTitle>The post will be visible to</SubTitle>
+        <Selector
```

Comment:
Let's update the styles to match Figma designs:

Current:
<img width="119" alt="Screen Shot 2020-05-23 at 1 27 07 AM" src="https://user-images.githubusercontent.com/42228386/82726589-176ae700-9c9a-11ea-8c79-90a8f8823091.png">

Figma:
<img width="196" alt="Screen Shot 2020-05-23 at 1 26 53 AM" src="https://user-images.githubusercontent.com/42228386/82726588-16d25080-9c9a-11ea-82ad-ff03985d043e.png">